internals/secrethub/signup.go

```diff
@@ -181,7 +185,37 @@ func (cmd *SignUpCommand) Run() error {
 	}
 
 	cmd.progressPrinter.Stop()
-
 	fmt.Fprintf(cmd.io.Stdout(), "Setup complete. To read your first secret, run:\n\n    secrethub read %s\n\n", secretPath)
+
+	createWorkspace := cmd.org != ""
+	if !createWorkspace {
+		createWorkspace, err = ui.AskYesNo(cmd.io, "Do you want to create a shared workspace for your team?", ui.DefaultYes)
+		if err != nil {
+			return err
+		}
+	}
+	if createWorkspace {
+		fmt.Fprint(cmd.io.Stdout(), "Please answer the questions below, followed by an [ENTER]\n\n")
+		if cmd.org == "" {
+			cmd.orgDescription, err = ui.AskAndValidate(cmd.io, "workspace name for your team (e.g. your company name): ", 2, api.ValidateOrgName)
```

Comment:
You're setting `cmd.orgDescription` here while it's the `cmd.org` field that needs to be set.

Reply:
oops